### DISCUSSION TITLE
Invitation emails are not sent

### DISCUSSION THREAD

**Discussion by somenotrandomcoder** [AntlerVC/firetable#1464] [Support / Q&A] [answered]
I tried to invite friend to project but invitation emails are not sent. There is also no link in UI i can just copy paste and send him myself. Trie with my second email - didnt receive invitation either.

**Comment by BeeBombshell** [comment]
Hey @somenotrandomcoder, invitation emails might sometimes end up in the promotions folder on your email. Could you perhaps try searching your inbox with keywords “rowy workspace” and see if you find it?

**Comment by Chadha93** [comment, marked as answer]
Marking this as answer, followed up here - https://github.com/rowyio/rowy/issues/1465#issuecomment-1791056632.

@somenotrandomcoder Feel free to reach out on [Discord community support](https://buildship.com/discord) if the problem still exists.
